product/charts/layouts/daily_tag_charts/AvailabilityZone.yaml

```diff
@@ -13,6 +13,17 @@
   :max_value: 100.4
   :decimals: 1
 
+- :title: Memory (MB)
+  :type: Line
+  :columns:
+  - derived_memory_used
+  :menu:
+  - Chart-Current-Hourly:Hourly on this day
+  - Chart-VMs-topday:Top Instances for <cat> on this day
+#  - Timeline-Current-Daily:Daily events in this Availability Zone
+#  - Timeline-Current-Hourly:Hourly events in this Availability Zone
```

Comment:
Alternatively, I could remove all of these as part of this PR -- but the point is if I don't add them here they should be removed elsewhere at the same time.